data/lib/core/player.lua

```diff
@@ -674,3 +664,47 @@ function Player.getTotalDefense(self)
 	end
 	return total
 end
+
+local blessStatusBits = {
+	[1] = BLESS_TYPE_WISDOM_OF_SOLITUDE,
+	[2] = BLESS_TYPE_SPARK_OF_PHOENIX,
+	[3] = BLESS_TYPE_FIRE_OF_THE_SUNS,
+	[4] = BLESS_TYPE_SPIRITUAL_SHIELD,
+	[5] = BLESS_TYPE_THE_EMBRACE_OF_THE_WORLD
+}
+
+function Player.getBlessingCount(self)
+	local blessings = 0
+	local bits = 0
+	for i = 1, SERVER_BLESSINGS_COUNT, 1 do
```

Comment:
![image](https://github.com/otland/forgottenserver/assets/28090948/2fb2361d-c0a7-4f68-905b-4ecac72f3619)
this 1 is unnecessary, this third parameter is only used when we want to use a number other than 1, since it is 1 by default

```lua
for i = 1, SERVER_BLESSINGS_COUNT, 1 do
-- The same:
for i = 1, SERVER_BLESSINGS_COUNT do
```